app/templates/main/userProfile.html

```diff
@@ -294,6 +295,112 @@ <h6>{{bgType.description}}</h6>
     {% endif %}
     <!-- End Background Check Table -->
 </div>
+<!-- Start Notes section -->
+  <div class="accordion-item">
+    <h3 class="accordion-header" id="headingSix">
+      {% set focus = "open" if visibleAccordion == "notes" else "collapsed" %}
+      <button class="accordion-button {{focus}}" type="button" data-bs-toggle="collapse" data-bs-target="#collapseSix" aria-expanded="false" aria-controls="collapseSix">
+          Notes
+      </button>
+    </h3>
+    {% set show = "show" if visibleAccordion == "notes" else "" %}
+    <div id="collapseSix" class="accordion-collapse collapse {{show}}" aria-labelledby="headingSix" data-bs-parent="#userProfile">
```

Comment:
Once you fix the page reload in the js so that it reloads on the accordion you will need to fix the focusing and opening of the accordions.
![image](https://user-images.githubusercontent.com/26262045/194913133-0f1dd192-10d5-49b4-90ef-56f79800c4c5.png)

Reply:
Fixed